src/BsReactstrap__Alert.re

```diff
@@ -1,44 +1,57 @@
 [@bs.module "reactstrap"] external alert : ReasonReact.reactClass = "Alert";
 
-[@bs.obj]
-external makeProps : (
-  ~className: string=?,
-  ~closeClassName: string=?,
-  ~closeAriaLabel: string=?,
-  ~cssModule: 'a=?,
-  ~color: string=?,
-  ~isOpen: bool=?,
-  ~toggle: 'b=?,
-  ~tag: 'c=?,
-  ~transition: 'd=?,
-  unit
-) => _ = "";
+type toggle;
+type transition;
 
-let make = (
-  ~className=?,
-  ~closeClassName=?,
-  ~closeAriaLabel=?,
-  ~cssModule=?,
-  ~color=?,
-  ~isOpen=?,
-  ~toggle=?,
-  ~tag=?,
-  ~transition=?,
-  children
-) =>
+[@bs.deriving abstract]
+type props = {
+  [@bs.optional]
+  className: string,
+  [@bs.optional]
+  closeClassName: string,
+  [@bs.optional]
+  closeAriaLabel: string,
+  [@bs.optional]
+  cssModule: BsReactstrap__Props.cssModule,
+  [@bs.optional]
+  color: string,
+  [@bs.optional]
+  isOpen: bool,
+  [@bs.optional]
+  toggle,
+  [@bs.optional]
+  tag: BsReactstrap__Props.tag,
+  [@bs.optional]
+  transition,
+};
+
+let make =
+    (
+      ~className=?,
+      ~closeClassName=?,
+      ~closeAriaLabel=?,
+      ~cssModule=?,
+      ~color=?,
+      ~isOpen=?,
+      ~toggle=?,
+      ~tag=?,
+      ~transition=?,
+      children,
+    ) =>
   ReasonReact.wrapJsForReason(
     ~reactClass=alert,
-    ~props=makeProps(
-      ~className?,
-      ~closeClassName?,
-      ~closeAriaLabel?,
-      ~cssModule?,
-      ~color?,
-      ~isOpen?,
-      ~toggle?,
-      ~tag?,
-      ~transition?,
-      ()
-    ),
-    children
-  );
+    ~props=
+      props(
+        ~className?,
+        ~closeClassName?,
+        ~closeAriaLabel?,
+        ~cssModule?,
+        ~color?,
+        ~isOpen?,
+        ~toggle?,
+        ~tag?,
+        ~transition?,
+        (),
+      ),
+    children,
+  );
```

Comment:
Can you insert final new lines for every file? If you are using vscode you can do it by adding `"files.insertFinalNewline": true` to workspace or user settings

Reply:
Will do. Hmmm I wonder what happened to my vsCode settings recently.